code/__defines/items_clothing.dm

```diff
@@ -148,7 +148,7 @@
 #define BODYTEMP_HEATING_MAX            30  // The maximum number of degrees that your body can heat up in 1 tick,   when in a hot  area.
 
 #define BODYTEMP_HEAT_DAMAGE_LIMIT 360.15 // The limit the human body can take before it starts taking damage from heat.
-#define BODYTEMP_COLD_DAMAGE_LIMIT 260.15 // The limit the human body can take before it starts taking damage from coldness.
+#define BODYTEMP_COLD_DAMAGE_LIMIT 273.15 // The limit the human body can take before it starts taking damage from coldness.
```

Comment:
See above, 0C weather won't even bring you near the *DANGER ZOOOONE* and even then the damage on the first threshold stabilizes in around 1 point of Burnloss, at least relatively short-term. Also those defines literally do nothing since species data are in, I believe :^)

I didn't want to edit heat threshold because I wasn't sure about balance side-effects with fires and such or I'd lower it too.